kimchi/js/bindings.js

```diff
@@ -2135,7 +2149,7 @@ var caml_lookup_patterns_to_rust = function (wasm_lookup_patterns, klass) {
   var range_check = wasm_lookup_patterns[3];
   var foreign_field_mul = wasm_lookup_patterns[4];
 
-  return new plonk_wasm.LookupPatterns(
+  return new plonk_wasm.WasmLookupPatterns(
```

Comment:
this constructor previously didn't exist. It's now added to the Rust side along with two others